src/git-lfs/install.sh

```diff
@@ -64,15 +60,51 @@ find_version_from_git_tags() {
     echo "${variable_name}=${!variable_name}"
 }
 
+# Get the list of GPG key servers that are reachable
+get_gpg_key_servers() {
+    declare -A keyservers_curl_map
+    keyservers_curl_map["hkp://keyserver.ubuntu.com"]="http://keyserver.ubuntu.com:11371"
+    keyservers_curl_map["hkp://keyserver.ubuntu.com:80"]="http://keyserver.ubuntu.com"
+    keyservers_curl_map["hkps://keys.openpgp.org"]="https://keys.openpgp.org:443"
+    keyservers_curl_map["hkp://keyserver.pgp.com"]="http://keyserver.pgp.com:11371"
```

Comment:
```suggestion
    keyservers_curl_map["hkp://keyserver.ubuntu.com"]="http://keyserver.ubuntu.com"
    keyservers_curl_map["hkp://keyserver.ubuntu.com:80"]="http://keyserver.ubuntu.com:80"
    keyservers_curl_map["hkps://keys.openpgp.org"]="https://keys.openpgp.org"
    keyservers_curl_map["hkp://keyserver.pgp.com"]="http://keyserver.pgp.com"
```

We have seen issues where the default ports (eg. 11371) are blocked. Hence, can we have the http URLs resemble the values we currently have?

Reply:
Default port for `hkp://` is 11371. Hence, I'm using this map to store `http://` equivalent destinations to be used with `curl`, as `curl` does not accept `hkp(s)://` addresses as destinations.